src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Pages/Account/Register.razor

```diff
@@ -29,7 +29,7 @@
             <DataAnnotationsValidator />
             <h2>Create a new account.</h2>
             <hr />
-            <ValidationSummary class="text-danger" />
```

Comment:
@surayya-MS There are a bunch of other instances of `<ValidationSummary class="text-danger" />`. I changed all of them in [the release/8.0 merge commit in my template update PR](https://github.com/dotnet/aspnetcore/pull/51134/commits/2350856c2bd43f398050b023adfd77ffed9c4aa9). Let me know if there was a particular reason for leaving the other instances out.

Reply:
Great! No particular reason, other that I overlooked those instances when focusing on fixing the filed bug. Thank you!